java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/Server.java

```diff
@@ -961,6 +966,7 @@ protected LanguageClient client() {
             ((LanguageClientAware) getTextDocumentService()).connect(client);
             ((LanguageClientAware) getWorkspaceService()).connect(client);
             ((LanguageClientAware) treeService).connect(client);
+            LSPServerTelemetryFactory.getDefault().connect(client);
```

Comment:
This will probably not work well with the `global` storage for NBLS, when multiple clients (vscode windows) share a NBLS instance.

Reply:
Fixed to work with multiple clients.